src/client/driver/driver.js

```diff
@@ -118,6 +118,7 @@ import createErrorCtorCallback, {
 import './command-executors/action-executor/actions-initializer';
 import { shouldSkipJsError } from './process-skip-js-errors';
 
+
```

Comment:
Removed please

Reply:
We usually add 2 new lines after imports.
So, this new line here is valid.